django_codemod/visitors/crypto.py

```diff
@@ -0,0 +1,22 @@
+from typing import Sequence
+
+from libcst import Arg, Call, Integer
+
+from django_codemod.constants import DJANGO_3_0, DJANGO_4_0
+from django_codemod.visitors.base import BaseFuncRenameTransformer
+
+
+class GetRandomStringTransformer(BaseFuncRenameTransformer):
+    """Update lengthless uses of get_random_string()"""
+
+    deprecated_in = DJANGO_3_0
```

Comment:
I think it's actually been deprecated in 3.1, no?

https://docs.djangoproject.com/en/dev/releases/3.1/#deprecated-features-3-1

Reply:
Oop, copy-paste typo. Good catch.